### ISSUE TITLE
use of "appcenter test run uitest ... --merge-nunit-xml blah.xml" results in error

### ISSUE THREAD

**Issue by jmegner** [microsoft/appcenter-cli#1208]
If I run the following command, it works and I get a nunit_xml_zip.zip containing a nunit report xml file for each device:
```
appcenter test run uitest `
    --app "JacobEgnerDemos/XamarinPipelineDemo" `
    --app-path (gci -Recurse "..\*Signed.apk")[0].FullName `
    --devices "JacobEgnerDemos/two_devices" `
    --test-series "master" `
    --locale "en_US" `
    --build-dir "..\XamarinPipelineDemo.UITest\bin\Release" `
    --uitest-tools-dir "..\XamarinPipelineDemo.UITest\bin\Release" `
    --test-output-dir "." `
```

If I add  `--merge-nunit-xml "AppCenterUiTestResult.xml"` to the command, I still get the zip file containing device-specific xml files but I also get this error message:
```
Error: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Document

Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and opening a support ticket (using the help menu in the upper right corner of the screen)


    Environment: win32
    App Upload Id: JacobEgnerDemos/XamarinPipelineDemo
    Timestamp: 1610245258160
    Operation: RunUITestsCommand
    Exit Code: 3
    User Email: [redacted]
    User Name: JacobEgner
    User Id: [redacted]
```

I was hoping the `--merge-nunit-xml` argument would get me a single merged xml file.

**Comment by Naweap** [comment]
Got exactly the same issue !

**Comment by AnatolyPristensky** [comment]
Hello @Naweap,
I've contacted appropriate team and bug has been confirmed. 
We will update status of this issue once get results.

**Comment by AnatolyPristensky** [comment]
Fixed in release [2.10.9](https://github.com/microsoft/appcenter-cli/releases/tag/v2.10.9)
